packages/kbn-test/src/auth/saml_auth.test.ts

```diff
@@ -41,9 +41,13 @@ const mockGetOnce = (mockedUrl: string, response: any) => {
 };
 
 describe('saml_auth', () => {
+  const attemptsCount = 3;
```

Comment:
@wayneseymour I'm not about it: one test in the suite uses `attemptsCount = 3`, the other one `attemptsCount = 2` - on purpose, to make sure it actually changes :)

Reply:
it is also applicable only to `describe('createCloudSession'` as we retry only for Cloud call

Reply:
ah cool, I'll back it out

Reply:
backed out: https://github.com/elastic/kibana/pull/186626/commits/f33fd74bacfdd4f5b471e013057057a05a1c35be